.github/issue_template.md

```diff
@@ -0,0 +1,21 @@
+# Issue type
+- Camayoc bug
+- Enhancement
+- Other
+
+# Description of issue
+
+# Completion checklist
+To close this issue, the following items must be satisfied:
+- [ ] One
+- [ ] Two
+
+# Related development issues
+
+### User stories
```

Comment:
Here you are using level 3 header, to make more meaningful use either level 2 or update the other headers to have all upper levels before.